kotlinx-coroutines-core/jvm/test/guide/example-cancel-02.kt

```diff
@@ -2,17 +2,18 @@
 package kotlinx.coroutines.guide.exampleCancel02
 
 import kotlinx.coroutines.*
+import kotlin.time.Duration.Companion.milliseconds
 
 fun main() = runBlocking {
-    val startTime = currentTimeMillis()
+    val startTime = markNow()
```

Comment:
Are these tests supposed to be executed with an adjustable time source?
Maybe we should just use `TimeSource.Monotonic` instead?

Reply:
Hmm, it seems to be generated by Knit, which replaces `System.currentTimeMillis()` with `currentTimeMillis()`: https://github.com/Kotlin/kotlinx-knit/blob/d461cde5558a8f9a9338f80fda29607fd3779b3b/src/Knit.kt#L205